raster/r.terraflow/r.terraflow.md

```diff
@@ -165,16 +165,21 @@ r.terraflow elev=elevation.10m filled=elevation10m.filled \
 
 ## REFERENCES
 
-1. The [TerraFlow](http://www.cs.duke.edu/geo*/terraflow/) project at Duke University
-2. [I/O-efficient algorithms for problems on grid-based
-    terrains](http://www.cs.duke.edu/geo*/terraflow/papers/alenex00_drainage.ps.gz). Lars Arge, Laura Toma, and Jeffrey S. Vitter. In
+1. The [TerraFlow](http://www.cs.duke.edu/geo*/terraflow/)
+   project at Duke University
+2. [I/O-efficient algorithms for problems on grid-based terrains]
+   (<http://www.cs.duke.edu/geo*/terraflow/papers/alenex00_drainage.ps.gz>).
+   Lars Arge, Laura Toma, and Jeffrey S. Vitter. In
     *Proc. Workshop on Algorithm Engineering and Experimentation*, 2000.
     To appear in *Journal of Experimental Algorithms*.
-3. [Flow computation on massive grids](http://www.cs.duke.edu/geo*/terraflow/papers/acmgis01_terraflow.pdf). Lars
+3. [Flow computation on massive grids]
+   (<http://www.cs.duke.edu/geo*/terraflow/papers/acmgis01_terraflow.pdf>). Lars
```

Comment:
Same broken link

Reply:
This is the DOI: https://doi.org/10.1145/512161.512180

and the PDF: https://web.archive.org/web/20170813084136/https://cs.duke.edu/geo*/terraflow/papers/acmgis01_terraflow.pdf

Reply:
The syntax of the link I meant, not what it pointed to

Reply:
I also replaced the links that were not working (or removed if there was no alternative)